modules/virt-example-nmstate-multiple-interfaces.adoc

```diff
@@ -7,26 +7,53 @@
 
 You can create multiple interfaces in the same node network configuration policy. These interfaces can reference each other, allowing you to build and deploy a network configuration by using a single policy manifest.
 
-The following example snippet creates a bond that is named `bond10` across two NICs and a Linux bridge that is named `br1` that connects to the bond.
+The following example snippet creates a bond that is named `bond10` across two NICs and VLAN that is named `bond10.103` that connects to the bond.
```

Comment:
```suggestion
The following example YAML file creates a bond that is named `bond10` across two NICs and VLAN that is named `bond10.103` that connects to the bond.
```

Reply:
@snarayan-redhat changing from "snippet" to "YAML file" eliminates ambiguity and continues the term used in the prior YAML example.